beacon-chain/types/block.go

```diff
@@ -271,6 +272,20 @@ func (b *Block) isAttestationValid(attestationIndex int, db beaconDB, aState *Ac
 		return false
 	}
 
+	// Retrieve block from db using justified_block_hash and verify its slot number is the same as attestation justified slot number.
+	rBlock, err := db.GetBlock(hash)
+	if err != nil {
+		log.Errorf("Failed to get block: %v", err)
+		return false
+	}
+	if attestation.JustifiedSlot != rBlock.SlotNumber() {
+		log.Errorf("The attestation's slot number is not the same as the slot number of the block retrieved by the justified slot hash. Attestation slot number: %v",
+			attestation.JustifiedSlot)
+		log.Errorf("Retrieved block slot number: %v",
```

Comment:
This log is short enough to be put in one line, it doesnt have to be split into two.

Reply:
Also the formatting for slot number should be `%d` not `%v`, the same applies in the log above